types/payments.ts

```diff
@@ -266,6 +266,11 @@ export interface AchReceivedPayment {
          */
         returnReason?: string
 
+        /**
+         * The direction in which the funds flow.
+         */
+        direction: Direction
```

Comment:
Let's add this as part of Pick<BasePaymentAttributes>

` } & Pick<BasePaymentAttributes, "createdAt" | "amount" | "direction" | "description" | "tags">`

maybe even:
` } & Omit<BasePaymentAttributes, "status" | "reason">`

Reply:
👍 that's cleaner